.travis.yml

```diff
@@ -26,4 +26,4 @@ matrix:
     - rvm: jruby-head
     - rvm: 1.9.3
 
-script: "rake compile && bundle exec rspec --color --backtrace --tag ~unfinished --seed 1 --format documentation ./spec"
+script: "rake compile && bundle exec rspec --color --backtrace --tag ~unfinished --seed 1 ./spec"
```

Comment:
I'm not sure if there's another way, but `--format documentation` has been in .travis.yml since [June of last year](https://github.com/ruby-concurrency/concurrent-ruby/commit/8672425108802baf816cd004f94b58c8f4c11191#diff-354f30a63fb0907d4ad57269548329e3). I only removed it a couple of weeks ago--then immediately added it back when we had a hanging test. I think we should keep it for now. Once we stabilize our tests we can remove it, but the full output is pretty valuable right now.